src/Services/LDAPAuth.php

```diff
@@ -161,7 +160,12 @@ protected function ldapOpen($username, $password)
                 $this->utils->createPasswordlessUserWithDefaultObjects($username, $displayName, $email);
 
                 $em = $this->doctrine->getManager();
-                $em->flush();
+
+                try {
+                    $em->flush();
+                } catch (\Exception $e) {
+                    error_log('LDAP Error (flush): '.$e->getMessage());
+                }
```

Comment:
Catching the exception will allow to continue (as the auth succeeded) and not log the trace, if any